dali/python/nvidia/dali/pipeline.py

```diff
@@ -152,14 +152,32 @@ def create_pipeline():
     by decorating them with `@dali.pickling.pickle_by_value`. It may be especially useful when
     working with Jupyter notebook to work around the issue of worker process being unable to import
     the callback defined as a global function inside the notebook.
+`output_dtype` : nvidia.dali.types or list of those, default = None
+    With this argument, you may declare, what data type you expect in the given output. You shall
+    pass a list of values from module:`nvidia.dali.types`, each element in the list corresponding to
```

Comment:
```suggestion
    pass a list of types.DALIDataType, each element in the list corresponding to
```
And please link to that type. Also adjust it in all the occurrences here in the doc and in the error messages. It can't be types.SampleInfo right?

Reply:
Done